census/src/test/scala/grasshopper/census/util/Generators.scala

```diff
@@ -0,0 +1,36 @@
+package grasshopper.census.util
+
+import org.scalacheck.{ Arbitrary, Gen }
+
+trait Generators {
+
+  case class Alphanumeric(s: String) {
+    override def toString = s.toString
+  }
+
+  case class HyphenStr(s: String) {
+    override def toString = s.toString
+  }
+
+  val unicodeChar = Gen.choose(Char.MinValue, Char.MaxValue).filter(c => !Character.isDigit(c))
+
+  val alphanumericGen: Gen[Alphanumeric] = {
+    for {
+      x <- Gen.oneOf("A", "B", "C")
+      y <- Gen.choose(Int.MinValue, Int.MaxValue)
+    } yield Alphanumeric(List(x, y).mkString)
+  }
+
+  implicit def arbAlphanumeric: Arbitrary[Alphanumeric] = Arbitrary(alphanumericGen)
+
+  val hyphenStrGen: Gen[HyphenStr] = {
+    for {
```

Comment:
In fact, looks like it definitely works that way.  Here's an example from the Travis run:

* https://travis-ci.org/cfpb/grasshopper/builds/77498549#L7872

Also, looks like the combination of all these failing parses in the test, and changing the logging to include the full exception overruns Travis's [10000 line limit](https://travis-ci.org/cfpb/grasshopper/builds/77498549#L10000).  Probably want to fix some aspect of this.